### PR TITLE
v3.0.0

### PR DESCRIPTION
**Pull request by gabelerner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,13 @@ DO NOT TOUCH. SAVE IT ON TOP.
 - JavaScript -> TypeScript.
 
 ### Fixed
+- [#782](https://github.com/canvg/canvg/issues/782)
+- [#750](https://github.com/canvg/canvg/issues/750)
+- [#735](https://github.com/canvg/canvg/issues/735)
+- [#406](https://github.com/canvg/canvg/issues/406)
+- [#757](https://github.com/canvg/canvg/issues/757)
+- [#748](https://github.com/canvg/canvg/issues/748)
+- [#694](https://github.com/canvg/canvg/issues/694)
```
Comment (gabelerner) [inline CHANGELOG.md]: @dangreen anything i missed?

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ JavaScript SVG parser and renderer on Canvas. It takes the URL to the SVG file o
 
 [Demo](https://canvg.github.io/canvg/demo/index.html)
 
-[Playground](https://jsfiddle.net/mdrv9qku/)
+[Playground](https://jsfiddle.net/5n230gh7/)
```
Comment (gabelerner) [inline README.md]: this link points to where 3.0 will generate

**Comment by dangreen** [comment]
@gabelerner ?

**Comment by gabelerner** [comment]
was afk today (daughter was sick) - will push this out tmrw and make a show HN post to market it a bit

**Pull request merged**